### ISSUE TITLE
Serializable io.sentry.event.interfaces.DebugMetaInterface$DebugImage

### ISSUE THREAD

**Issue by pierceshih** [getsentry/sentry-java#549]
Using 1.6.4 sentry-android library with Android API 25 and Sentry's gradle plugin, attempting to buffer events to disk before sending yields the following error:

```
Error writing Event to offline storage: <uuid>
java.io.NotSerializableException: io.sentry.event.interfaces.DebugMetaInterface$DebugImage
at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1224)
...
more java.io.** stack traces abridged
...
at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:346)
at io.sentry.buffer.DiskBuffer.add(DiskBuffer.java:79)
at io.sentry.connection.BufferedConnection$2.send(BufferedConnection.java:169)
at io.sentry.SentryClient.sendEvent(SentryClient.java:134)
at io.sentry.SentryClient.sendEvent(SentryClient.java:175)
at io.sentry.Sentry.capture(Sentry.java:177)
...
```

It seems like `DebugImage` would need to implement `java.io.Serializable`? Please advise.

**Comment by bretthoerner** [comment]
Ack, you're right. I'll fix and release 1.6.5.

**Comment by bretthoerner** [comment]
https://github.com/getsentry/sentry-java/releases/tag/v1.6.5

Syncing to Maven mirrors now, let me know if it doesn't do the trick!
